src/observer.ts

```diff
@@ -21,19 +21,19 @@ import got, { Got, RequestError, Response } from 'got';
 import crypto from 'node:crypto';
 import pMap from 'p-map';
 
+import { EpochTimestampSource } from './protocol.js';
 import {
   ArnsNameAssessment,
   ArnsNameAssessments,
   ArnsNamesSource,
   EntropySource,
-  EpochHeightSource,
   GatewayAssessments,
   GatewayHostsSource,
   ObserverReport,
   OwnershipAssessment,
 } from './types.js';
 
-const REPORT_FORMAT_VERSION = 1;
+export const REPORT_FORMAT_VERSION = 2;
```

Comment:
@djwhitt - is the report format intended to reflect the `assessments`. I've changed the top level attributes to include `epochStartTimestamp` and `epochEndTimestamp`. can revert it back to 1 if that's acceptable.

Reply:
Yeah, you're doing the right thing. We should bump the version for this.